modules/jdbc/src/main/java/org/apache/ignite/internal/jdbc/JdbcStatement.java

```diff
@@ -432,10 +438,14 @@ public boolean getMoreResults() throws SQLException {
     public boolean getMoreResults(int curr) throws SQLException {
```

Comment:
don't know where to put this, so let it be here:

for sequence of DML, expression `!stmt.getMoreResults() && stmt.getUpdateCount() == -1` never returns `true` (that is now we determine there are no more results, see javadoc for `getMoreResults`)

Reply:
fixed